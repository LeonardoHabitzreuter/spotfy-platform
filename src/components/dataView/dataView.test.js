import React from 'react'
import { shallow } from 'enzyme'
import DataViewList from './index'

describe('DataViewList component', () => {
  const wrapper = shallow(
    <DataViewList><label>Test</label></DataViewList>
  )

  test('should render properly', () => {
    expect(wrapper).toMatchSnapshot()
  })

  test('should render children', () => {
    expect(wrapper.exists('label')).toBeTruthy()
  })

  test('should contains a reference to the item component', () => {
    expect(DataViewList.Item).not.toBeUndefined()
  })
})
