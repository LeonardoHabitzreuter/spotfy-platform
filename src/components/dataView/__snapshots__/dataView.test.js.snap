// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DataViewList component should render properly 1`] = `
<ul
  className="list-group"
>
  <label>
    Test
  </label>
</ul>
`;
